packages/wonder-blocks-tooltip/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "private": true,
+  "name": "@khanacademy/wonder-blocks-tooltip",
+  "version": "0.0.1",
+  "design": "v1",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "",
+  "main": "dist/index.js",
+  "source": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@khanacademy/wonder-blocks-color": "^1.0.0",
+    "@khanacademy/wonder-blocks-core": "^1.0.0",
+    "@khanacademy/wonder-blocks-modal": "^0.0.3",
+    "@khanacademy/wonder-blocks-typography": "^1.0.0"
```

Comment:
These should match the current version, e.g. `"@khanacademy/wonder-blocks-color": "^1.0.2"` (or whatever it is).  If it isn't the same, then `lerna` won't bump the version automatically.  Probably not a huge deal, but some of the other packages are doing this so we should be consistent.

Reply:
Yeah, I think the versions changed since I made the PR. I'll make sure to refresh them. Thanks for bringing this up. 

I wonder, going forward, how we can ensure this is the case when we merge code. Perhaps there's a script we can add to our tests that validates the versions in new PRs?